### PR TITLE
Readme for remote inbox notifications

### PR DESCRIPTION
**Pull request by becdetat**
This adds a readme for the remote inbox notifications system.

### CODE REVIEW

File: src/RemoteInboxNotifications/README.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+	"index": "revenue",
+	"operation": "=",
+	"value": "none"
+}
+```
+
+`index`, `operation`, and `value` are all required.
+
+### Is eCommerce
+This passes when the store is on the eCommerce plan.
```
Comment (mattsherman) [inline src/RemoteInboxNotifications/README.md:334]:
```suggestion
This passes when the store is on a WordPress.com site with the eCommerce plan.
```

**Review by mattsherman — APPROVED** [review]
It's great to have this documentation in the codebase! Nice work.

I left a few suggestions, but have pre-approved this so that you can merge it after you make the changes (or, decide that the changes should not be made).

File: src/RemoteInboxNotifications/README.md [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+},
+{
+	"name": "set-up-concierge",
+	"locales": [
+		{
+			"locale": "en_US",
+			"label": "Schedule free session"
+		}
+	],
+	"url": "https:\/\/wordpress.com\/me\/concierge",
```
[the remark this answers is not in the document]
Comment (becdetat) [reply src/RemoteInboxNotifications/README.md:107]: not sure how the escaping got in here

**Comment by becdetat** [comment]
Thanks @mattsherman :-)

**Pull request merged**